### PR TITLE
RFE: add [Back] button to the Associate bug screen

### PR DESCRIPTION
**Pull request by Riju19**

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

**Comment by gayathrirajendar** [comment]
Hey @Riju19 ,

While your contribution is reviewed by admins, If you get a chance, please read [this post](https://chris.beams.io/posts/git-commit/ ) on commit messages. 

It's a very nice read suggested by the mentor. Also, you can look at the commit messages of this repository which follows a pattern.

Happy e-meeting you! 
Thanks.

**Comment by Riju19** [comment]
@gayathrirajendar All right, I will! Thanks.

**Comment by marusak** [comment]
Thank you @Riju19 for the PR :)
Can you please update the commit message as suggested by @gayathrirajendar ?
Thank you

**Comment by Riju19** [comment]
@marusak @gayathrirajendar Yes, just changed it. I hope it's better now!

File: src/webfaf/templates/reports/associate_bug.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {% block body %}
   <div class="row">
     <div class="col-md-12">
-      <p>Associate bug with report {{report.id}}</p>
+      <p>Associate bug with report <a href=".">{{report.id}} <a></p>
```
Comment (marusak) [inline src/webfaf/templates/reports/associate_bug.html]: There is not end tag, only two opening ones.

**Comment by Riju19** [comment]
@marusak I made the changes.

**Comment by marusak** [comment]
Thank you!
Closed by https://github.com/abrt/faf/commit/8baa4cdb0487110092451c679a86269114480ad1

**Pull request closed without merging**